dev-python/markuppy/markuppy-1.14-r1.ebuild

```diff
@@ -17,8 +17,13 @@ SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
 S="${WORKDIR}/${MY_P}"
 
 SLOT="0"
-LICENSE="BSD-2"
+LICENSE="MIT"
 KEYWORDS="~amd64"
 
 # MarkupPy does not have any test suite
 RESTRICT="test"
+
+src_prepare() {
+	eapply_user
+	sed -i -e 's/description-file/description_file/' setup.cfg || die "Failed to fix setup.cfg"
```

Comment:
This needs to be fixed upstream, not patched in Gentoo.

Reply:
I know, in fact I've opened a PR upstream but seems dead since 2017